src/cls/_ZPM/PackageManager/Developer/Processor/CSPApplication.cls

```diff
@@ -1,431 +0,0 @@
-Include (%sySecurity, %occErrors)
-
-Class %ZPM.PackageManager.Developer.Processor.CSPApplication Extends (%ZPM.PackageManager.Developer.Processor.Abstract, %XML.Adaptor, %Installer.CSPApplication) [ PropertyClass = %ZPM.PackageManager.Developer.Processor.PropertyParameters ]
```

Comment:
Actually keep this for now and we will deprecate it because it is widely used despite it's side effects. Should be named %ZPM.ResourceProcessor.CSPApplication. Also update in spreadsheet once done.